DataModel/CallToContainer.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+
+namespace ChameleonSystem\UpgradeHelperBundle\DataModel;
+
+/**
+ * CallToContainer represents a call to the `get` method on the symfony container
+ */
+final class CallToContainer
+{
+    /** @var string Explicit call using a string as argument. eg. $container->get('foo') */
```

Comment:
symfony code style guidelines explicitly state, that every  docblock should be multiline: 

> Don’t inline PHPDoc blocks, even when they contain just one tag (e.g. don’t put `/** {@inheritdoc} */` in a single line);

Reply:
sad.. I've added the whitespace

Reply:
:ok: